package.json

```diff
@@ -41,7 +41,10 @@
     "tslint-config-prettier": "^1.18.0",
     "typescript": "^3.8.3"
   },
-  "dependencies": {},
+  "dependencies": {
```

Comment:
I am not sure it is a good idea to add lodash as a dependency at this point. I think it is better to keep this dependency free users other than Toolio.

We are using entries and memoize from lodash.
For entries, can we use Object.entries?
For memoize, grabbing lodash source may also work, as it is tiny. 
```
function memoize(func, resolver) {
    const memoized = function (...args) {
        const key = resolver ? resolver.apply(this, args) : args[0];
        const cache = memoized.cache;

        if (cache.has(key)) {
            return cache.get(key);
        }
        const result = func.apply(this, args);
        memoized.cache = cache.set(key, result) || cache;
        return result;
    };
    memoized.cache = new Map();
    return memoized;
}
```

Reply:
Makes sense a lot and applied 😁 Thank you Cihan